src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+public class DukeException extends Exception{
+
+    protected enum ExceptionType {
+        INVALID_INPUT("\u2639 OOPS!!! I'm sorry, but I don't know what that means :-(\n"),
+        EMPTY_DESCRIPTION("\u2639 OOPS!!! The description of a task cannot be empty.\n"),
+        INDEX_OUT_OF_BOUND("\u2639 OOPS!!! No such task is found!\n"),
```

Comment:
Maybe can replace unicode with a constant that explains what it is?

Reply:
Hi Jia Yee, thanks for the comment, but I'm not sure what do you mean by constant. Isn't enum a special class consisting of constants? Would it be better if I use 
`public static final String INVALID_INPUT = "\u2639 OOPS!!! I'm sorry, but I don't know what that means :-(";`

Thanks.